### DISCUSSION TITLE
Corvus needs to run lunch command twice

### DISCUSSION THREAD

**Discussion by Stalick** [ROM-builders/temporary#10036] [General]
Corvus main dev said we need to run lunch command twice but there's a check on test.sh that prevents you from running that command twice. I already tried building it with just one lunch command but it didn't work.

**Comment by Apon77** [comment]
> Corvus main dev said we need to run lunch command twice but there's a check on test.sh that prevents you from running that command twice. I already tried building it with just one lunch command but it didn't work.

how to fix?

**Comment by Apon77** [comment]
> Corvus main dev said we need to run lunch command twice but there's a check on test.sh that prevents you from running that command twice. I already tried building it with just one lunch command but it didn't work.

try building corvus now with two lunch command, hope it will work

**Reply by Stalick** [reply]
> > Corvus main dev said we need to run lunch command twice but there's a check on test.sh that prevents you from running that command twice. I already tried building it with just one lunch command but it didn't work.
> 
> try building corvus now with two lunch command, hope it will work

It worked, thanks!
